scintellometry/meta/observations.py

```diff
@@ -61,17 +60,27 @@ def gmrt_rawfiles(file_fmt, fnbase, **kwargs):
     return a 2-tuple for GMRT observation 'key':
     (timestamp file, [file1, ...])
 
-    Number of files depends on whether there is a ``nodes`` keyword argument
+    Number of files depends on whether there is a ``feeds`` keyword argument
     """
-    nodes = kwargs.get('nodes', None)
-    if nodes is None:
+    from scintellometry.phasing.GMRTNaming import getNode,getVolt
+    
+    feeds = kwargs.get('feeds', None)
+
+    # Find raw_voltage and node numbers for given GMRT feeds
+    nodes = [getNode(feed) for feed in feeds]
```

Comment:
I'd make both `getNode` and `getVolt` dictionaries (see below; note also that python has the convention that function names are all lower-case, with parts separated by underscores; classes are CamelCase, including initial capital). So, something like
```
try:
    n_and_v = [nodes_and_voltages[feed] for feed in feeds]
except KeyError:
    raise KeyError("One or more of feeds '{0}' are not recognized as GMRT feeds"
                     .format(feeds)) 
```